features/305.Big_Data_Spark.feature

```diff
@@ -0,0 +1,133 @@
+Feature: test tutorial 305.Big Data (Spark)
+
+  This is the feature file of the FIWARE Step by Step tutorial for Big Data (Spark)
+  url: https://fiware-tutorials.readthedocs.io/en/latest/big-data-spark.html
+  git-clone: https://github.com/FIWARE/tutorials.Big-Data-Spark.git
+  git-directory: /tmp/tutorials.Big-Data-Spark
+  shell-commands: git checkout NGSI-v2 ; ./services create; ./services start
+  clean-shell-commands: ./services stop
+
+
+    Background:
+        Given I set the tutorial 305.Spark
+
+    # COMPILING A JAR FILE FOR SPARK --
+    Scenario: Compiling a Jar File For Spark
+        Given I prepare the script create-spark-jar.sh to be run
+        And   I set the environ variable WORKING_DIR to "cosmos-examples" under git
+        And   I set the environ variable OUTPUT_DIR to "cosmos-examples/target" under git
+        When  I exec the script as in the tutorial page
```

Comment:
When  I execute the script as in the tutorial page

Reply:
same as the previous one.

Reply:
same as the previous one.

Reply:
Then it should be I run the script as in the tutorial page

Reply:
It is "i runthe script"

Reply:
NTC